### PR TITLE
ProjectViewPopupMenu has Copy build-root relative path

### PR DESCRIPTION
**Pull request by illicitonion**

### CODE REVIEW

**Comment by baroquebobcat** [comment]
Could you add a couple screenshots of the new UI?

File: src/com/twitter/intellij/pants/ui/CopyPathRelativeToBuildRootAction.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Optional;
+
+public class CopyPathRelativeToBuildRootAction extends AnAction {
+  @Override
+  public void actionPerformed(AnActionEvent event) {
+    Optional<VirtualFile> maybeBuildRoot = PantsUtil.findBuildRoot(event.getProject());
+    Optional<VirtualFile> maybeFile = PantsUtil.getFileForEvent(event);
+    if (!maybeBuildRoot.isPresent() || !maybeFile.isPresent()) {
```
Comment (baroquebobcat) [inline src/com/twitter/intellij/pants/ui/CopyPathRelativeToBuildRootAction.java]: Could you DeMorgans this to `!(mBR.isPresent() && mF.isPresent())`. I think that'd be clearer and would mirror the expression used in `update`

Comment (illicitonion) [reply src/com/twitter/intellij/pants/ui/CopyPathRelativeToBuildRootAction.java]: I've never seen DeMorgan used as a verb before, I like it :D Done!

**Comment by wisechengyi** [comment]
Unrelated error. merged

**Pull request merged**